tests/utils.go

```diff
@@ -63,12 +63,12 @@ import (
 	"k8s.io/client-go/tools/remotecommand"
 
 	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/datavolumecontroller/v1alpha1"
-	"kubevirt.io/kubevirt/pkg/api/v1"
+	v1 "kubevirt.io/kubevirt/pkg/api/v1"
```

Comment:
redundant

Reply:
VSCode entered these explicit package names automatically once I saved each of these files, so I couldn't even remove them (unless I stopped using vscode).

Reply:
Resolved this issue in VSCode with the help of Meni.